tox.ini

```
@@ -8,6 +8,7 @@ isolated_build = true
 changedir = .tmp/{envname}
 deps =
     sphinx40: sphinx==4.0.0
```

you'll hate this bikeshedding, but what about:
Then we can just change that pin depending on the drop schedule, etc. 
```suggestion
    sphinx_oldest: sphinx==4.0.0
```